app/components/address-bar.js

```diff
@@ -0,0 +1,9 @@
+import Ember from "ember";
+
+export default Ember.Component.extend({
+  classNames: ["url-bar"],
+
+  change(event) {
+    this.sendAction("urlChanged", event.target.value);
+  }
```

Comment:
I'm not sure. But it seems that in testing, fillIn never fires the enter event, whatever you do.

http://stackoverflow.com/questions/28586314/trigger-an-inputs-action-from-an-acceptance-test#comment45530171_28596428